ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/plugin/GeneralInfoSection.java

```diff
@@ -354,19 +354,22 @@ public void refresh() {
 		// Select the corresponding 'Bundle-Shape' radio button (eventually)
 		Stream.of(fBundleShapeDefault, fBundleShapeJar, fBundleShapeDir).forEach(b -> b.setSelection(false));
 		Button selectedBundleShape = fBundleShapeDefault;
-		IManifestHeader header = getBundle().getManifestHeader(ICoreConstants.ECLIPSE_BUNDLE_SHAPE);
-		if (header != null) {
-			String value = header.getValue();
-			if (value != null) {
-				selectedBundleShape = switch (value) {
-					case ICoreConstants.SHAPE_JAR -> fBundleShapeJar;
-					case ICoreConstants.SHAPE_DIR -> fBundleShapeDir;
-					default -> null; // unsupported value
-				};
+		IBundle bundle = getBundle();
+		if (bundle != null) {
+			IManifestHeader header = getBundle().getManifestHeader(ICoreConstants.ECLIPSE_BUNDLE_SHAPE);
```

Comment:
You have a variable holding the result of getBundle(), so it's more consistent to use that here.

Reply:
Thank you, i did used it in my previous closed PR, this time i missed. i ll use the variable now.

Reply:
Done.